erpnext/controllers/stock_controller.py

```diff
@@ -226,7 +226,6 @@ def make_batches(self, warehouse_field):
 						doctype='Batch',
 						item=d.item_code,
 						supplier=getattr(self, 'supplier', None),
-						batch_naming_series=d.batch_naming_series,
```

Comment:
@meshramsaroj Why did you removed this. Its used for stock entry and purchase receipt
don't remove this